pkgs/applications/science/programming/scyther/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, callPackage, callPackage_i686, buildEnv
+, includeGUI ? true
```

Comment:
I don't know this package at all; how useful is it to its audience without the GUI? I'm wondering whether this should default to `false`.